src/main/java/uk/gov/companieshouse/authcodenotification/utils/Encrypter.java

```diff
@@ -31,35 +31,24 @@ public String encrypt(String plainTextToEncrypt, String key) throws EncryptionEx
         Validate.notNull(plainTextToEncrypt, "plainTextToEncrypt must not be null");
         Validate.notNull(key, "key must not be null");
 
-        byte[] keyBytes = key.getBytes(StandardCharsets.UTF_8);
-        byte[] plainTextBytes = plainTextToEncrypt.getBytes(StandardCharsets.UTF_8);
-
-        Cipher cipher;
-        byte[] initialisationVector;
         try {
-            cipher = Cipher.getInstance(AES_CBC_PKCS_5_PADDING);
+            byte[] keyBytes = key.getBytes(StandardCharsets.UTF_8);
+            byte[] plainTextBytes = plainTextToEncrypt.getBytes(StandardCharsets.UTF_8);
+
+            Cipher cipher = Cipher.getInstance(AES_CBC_PKCS_5_PADDING);
             var random = SecureRandom.getInstanceStrong();
-            initialisationVector = new byte[Cipher.getInstance(AES_CBC_PKCS_5_PADDING).getBlockSize()];
+            byte[] initialisationVector = new byte[Cipher.getInstance(AES_CBC_PKCS_5_PADDING).getBlockSize()];
             random.nextBytes(initialisationVector);
-        } catch (NoSuchAlgorithmException | NoSuchPaddingException e){
-            throw new EncryptionException(e.getMessage(), e);
-        }
 
-        try {
             cipher.init(Cipher.ENCRYPT_MODE, new SecretKeySpec(keyBytes, AES), new IvParameterSpec(initialisationVector));
-        } catch (InvalidAlgorithmParameterException e) {
-            throw new EncryptionException(e.getMessage(), e);
-        }
 
-        byte[] encodedBytes;
-        byte[] encodedSaltedBytes;
-        try {
-            encodedBytes = cipher.doFinal(plainTextBytes);
-            encodedSaltedBytes = concatByte(initialisationVector, encodedBytes);
-        } catch (IllegalBlockSizeException | BadPaddingException | IOException e) {
+            byte[] encodedBytes = cipher.doFinal(plainTextBytes);
+            byte[] encodedSaltedBytes = concatByte(initialisationVector, encodedBytes);
```

Comment:
Very minor, but can this variable be renamed to iVAndCiphertext - salt is a little misleading (my fault I know - sorry)